apps/jackbnb-fe/src/components/Form/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {
+    FormControl,
+    Input
+  } from "@chakra-ui/react"
+  import MyTestButtonStyles from './index.module.css'
```

Comment:
Corregir los espacios
```suggestion
} from "@chakra-ui/react"

import MyTestButtonStyles from './index.module.css'
```